figs/Scidata2019_figs_genomicVariants.Rmd

```diff
@@ -154,22 +137,21 @@ We were able to crosscheck other samples in the bank with their clinical sequenc
 ```{r sample 2-031, echo= FALSE, eval=TRUE, fig.height=8,fig.width=8, results='show', message=FALSE, warning=FALSE}
 
 #Select specific sample files (select only the normal tissue and MPNST tissue and cell line samples)
-sample_2031 <- subset(jhu_anno_new, assay == "exomeSeq" & individualID == "2-031" & specimenID != "2-031 Xenograft")
-
+sample_2031 <- jhu_public_clindata[(jhu_public_clindata$individualID == "2-031" & jhu_public_clindata$specimenID != "2-031 Xenograft"),]
 
 #Make Maf file for sample 2-031
-jhu_sample_2031 <- merge_mafs(c(sample_2031$filepath), verbose = TRUE, clinicalData = sample_2031) 
+jhu_sample_2031 <- subsetMaf(jhu_public_maf_all, tsb = c(sample_2031$Tumor_Sample_Barcode)) 
 
 # Make oncoplot to show the SNPs and mutation of the samples in various genes of interest
 print("Fig 1 : Sample 2-031")
 png("Sample_2-031.png", width=800, height = 800)
 oncoplot(jhu_sample_2031, genes = c("NF1","PIK3CA", "PRC2","MEK1", "MEK2", "SUZ12", "ASTE1", "EGFR", "LRP1B", "MSH3", "PDCD1", "S1PR2", "EED", "EZH2"),
```

Comment:
I'm not sure why the other sample names show up in this figure when they're not there. any ideas? we can remove manually if need be.

Reply:
Not quite sure... looking into it..